Dockerfile

```diff
@@ -0,0 +1,38 @@
+# escape=`
+
+RUN pip install pymongo
+RUN pip install Flask
+
+FROM microsoft/windowsservercore as msi

```

Comment:
Why are we using a windows image as a base? Does this even work outside of windows?